playbooks/roles/ocp-config/README.md

```diff
@@ -17,6 +17,7 @@ Role Variables
 | workdir                 | no       | ~/ocp4-workdir | Place for config generation and auth files  |
 | log_level               | no       | info           | Option --log-level in openshift-install cmd |
 | release_image_override  | no       | ""             | OCP image overide variable                  |
+| enable_local_registry   | no       | false          | Flag to be set to true to enable usage of local registry only in case of disconnected install |
```

Comment:
"Flag to be set to true to enable usage of local registry only in case of disconnected install" => "Set to true to enable usage of local registry for restricted network install"